servers/movie_writer/movie_writer_pngwav.cpp

```diff
@@ -66,7 +66,19 @@ Error MovieWriterPNGWAV::write_begin(const Size2i &p_movie_size, uint32_t p_fps,
 		base_path = "res://" + base_path;
 	}
 
-	{
+	if (GLOBAL_GET("editor/movie_writer/create_new_file_if_existing")) {
```

Comment:
This is exactly from the original [PR](https://github.com/godotengine/godot/pull/71420/files#diff-c37abec8cc36493081275313d9a3d08701b8d6dcf21fae116dc4f837247e28c1R69)